### ISSUE TITLE
Debuggin react app does not work

### ISSUE THREAD

**Issue by philipreimer** [firefox-devtools/vscode-firefox-debug#215]
Steps to reproduce:

1. create new react app: `npx create-react-app my-app`
1. start react app: `export BROWSER=none && yarn start`
1. start debugging in vscode - see `launch.json` below
1. set a breakpoint in App.js file
1. change some code in App.js file, e.g. add a `console.log('foo')`

Result:
after changing anything in App.js file no breakpoint is hit anymore

Expected Result:
all breakpoints should still be working after changing code

launch.json:
```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch localhost",
            "type": "firefox",
            "request": "launch",
            "reAttach": true,
            "url": "http://localhost:3000",
            "webRoot": "${workspaceFolder}/src"
        }
    ]
}
```

**Comment by sirlordt** [comment]
I had similar issue. The debug cannot start. Only create "Tab 1", "Tab 2", "Tab 3" .... "Tab N" in the Visual Studio Code -> Call Stack Panel. And the react web page only reload and reload and reload inside of firefox browser.

**Comment by philipreimer** [comment]
@sirlordt I think you have a different issue. In my case, debugging works correctly as long as I do not change any code.
Only when I change some code and is hot-reloaded in the dev server, breakpoints are not hit anymore.

**Comment by hbenl** [comment]
@philipreimer This is a long-standing bug in CRA (facebook/create-react-app#6074), see also [here](https://github.com/firefox-devtools/vscode-firefox-debug/issues/199#issuecomment-625377947).
I'll try to create a workaround for this.

**Comment by hbenl** [comment]
Try adding `"enableCRAWorkaround": true` to your settings and tell me if it works for you.

**Comment by philipreimer** [comment]
@hbenl `"enableCRAWorkaround": true` seems to help mostly.

What I noticed: I sometimes had to manually reload the page in the browser (F5) after hot-reload before breakpoints were hit.
When the page was reloaded automatically (triggered by hot-reload in the dev-server) breakpoints were not hit.

Another issue is that I can't seem to set any breakpoints inside arrow functions like the following (code sample from a react app in typescript):
```
1  const authors = props.comments.map(comment =>
2    comment.author
3  );
```

If I set a breakpoint in line "2" above it is ignored.

Breakpoint only works if I change the code to the following:
```
  const authors = props.comments.map(comment => {
    return comment.author;
  });
```

But that might be another issue because it happens also when restarting the dev-server, not only after hot-reloading.
